OrbitCore/Capture.h

```diff
@@ -25,23 +25,20 @@ class Capture {
  public:
   static void Init();
   static void SetTargetProcess(std::shared_ptr<Process> process);
```

Comment:
Related to my comment to removing the use of GTargetProcess in Capture.cpp, I think this method should dissappear as well if that is being deprecated

Reply:
Yes, it will be removed in a separate CL. Currently, I still maintain the global, but removed a couple of reads from it.